src/overlays/actors/ovl_En_Hs/z_en_hs.c

```diff
@@ -261,7 +261,7 @@ void func_8095345C(EnHs* this, GlobalContext* globalCtx) {
     } else if (this->actor.home.rot.x >= 20) {
         this->actionFunc = func_80953354;
         this->unk_2A2 = 40;
-    } else if CHECK_FLAG_ALL (this->actor.flags, ACTOR_FLAG_10000) {
+    } else if (CHECK_FLAG_ALL(this->actor.flags, ACTOR_FLAG_10000)) {
```

Comment:
There are so many of these 😩

Reply:
I honestly have no idea how I did that, it looks so bad